GNUmakefile

```diff
@@ -90,15 +90,17 @@ generate: abigen codecgen mockery ## Execute all go:generate commands.
 	go generate -x ./...
 
 .PHONY: testscripts
-testscripts: chainlink-test ## Install and run testscript against testdata/scripts/* files.
+testscripts: install-chainlink ## Install and run testscript against testdata/scripts/* files.
 	go install github.com/rogpeppe/go-internal/cmd/testscript@latest
-	go run ./tools/txtar/cmd/lstxtardirs -recurse=true | PATH="$(CURDIR):${PATH}" xargs -I % \
-		sh -c 'testscript -e COMMIT_SHA=$(COMMIT_SHA) -e HOME="$(TMPDIR)/home" -e VERSION=$(VERSION) $(TS_FLAGS) %/*.txtar'
+	go run ./tools/txtar/cmd/lstxtardirs -recurse=true | while read -r dir; do \
+		testscript -e COMMIT_SHA=$(COMMIT_SHA) -e HOME="$(TMPDIR)/home" -e VERSION=$(VERSION) $(TS_FLAGS) "$$dir"/*.txtar; \
+	done
```

Comment:
Will this work for everyone? We had a lot a trouble getting this settled in to a cross platform solution.

Reply:
hmm - the previous version does not work on mac
```
xargs: command line cannot be assembled, too long
```

This update should work for everyone

Reply:
cc @cedric-cordenier didn't this work before when you added `cmd/lstxtardirs` ?